chardet/constants.py

```diff
@@ -37,11 +38,3 @@
 eItsMe = 2
 
 SHORTCUT_THRESHOLD = 0.95
-
-import __builtin__
-if not hasattr(__builtin__, 'False'):
-    False = 0
-    True = 1
-else:
-    False = __builtin__.False
-    True = __builtin__.True
```

Comment:
This is strange. Wonder why this was something that was originally included...

Reply:
@rsiemens in reallllly old versions of Python there was not `True` or `False` builtin constants; the standard was to use `1` and `0`. That's the reason why `isinstance(True, int) and isinstance(False, int)` by the way.

Reply:
Ah TIL, thank you!